pkg/machine/qemu/machine.go

```diff
@@ -517,14 +514,22 @@ func (v *MachineVM) Start(name string, _ machine.StartOptions) error {
 	if err != nil {
 		return err
 	}
-
 	fd, err := qemuSocketConn.(*net.UnixConn).File()
 	if err != nil {
 		return err
 	}
 
+	dnr, err := os.Open("/dev/null")
+	if err != nil {
+		return err
+	}
+	dnw, err := os.Open("/dev/null")
+	if err != nil {
+		return err
+	}
+
 	attr := new(os.ProcAttr)
-	files := []*os.File{os.Stdin, os.Stdout, os.Stderr, fd}
+	files := []*os.File{dnw, dnr, dnr, fd}
```

Comment:
Should we make this conditional on debug level? So when debug we still attach stdout/err to get the qemu output. Otherwise the statement on line 540 is pretty useless.

Reply:
All of the output from qemu in this case ends up in its display (the qemu window), so nothing is actually lost.

Reply:
The fds are inverted. stdin should be dnr, and stdou/err should be dnw.